src/plugins/content/accessibility/keyNavigation/strategies/itemNavigation.js

```diff
@@ -90,7 +90,10 @@ export default {
          * @param {Object} cursor - The cursor definition supplied by the keyNavigator
          * @returns {jQuery} - The selected choice element
          */
-        const getQtiChoice = cursor => cursor && cursor.navigable.getElement().closest('.qti-choice');
+        const getQtiChoice = function (cursor){
+            console.log(cursor);
```

Comment:
Please remove `console.log`. I also saw stuff like `logger.debug` being used, but didn't check how to configure log level.

Reply:
My bad